### DISCUSSION TITLE
Embedding (TI) support?

### DISCUSSION THREAD

**Discussion by aolko** [lllyasviel/Fooocus#858] [Q&A] [answered]
Are you going to implement TI support? Writing negative word salad aain't fun and we already passed that point in a1111/InvokeAI

**Comment by frankyifei** [comment]
I have been using it everyday, use it in prompt just like comfyui such as (embedding:negativeXL_D)

**Comment by lllyasviel** [comment, marked as answer]
(embedding:filename:0.8)

**Reply by aolko** [reply]
Can i run it without a weight?

**Reply by AkoZ** [reply]
as just before answered, it seems ok without Weight too

**Reply by ykhasia** [reply]
by the way, regarding to the "(embedding:filename:0.8)", is it should be placed in Prompt input box ? or Negative Prompt input box?

![image](https://github.com/lllyasviel/Fooocus/assets/52735731/4ea95c91-6826-4474-9851-a1b9d1e69b2a)
